llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -10258,7 +10298,7 @@ static bool areRuntimeChecksProfitable(GeneratedRTChecks &Checks,
 
   // Skip vectorization if the expected trip count is less than the minimum
   // required trip count.
-  if (auto ExpectedTC = getSmallBestKnownTC(PSE, L)) {
+  if (auto ExpectedTC = getSmallBestKnownTC(PSE, CM.TheLoop)) {
```

Comment:
unrelated?

Reply:
Again, it is related because we don't have a `L` function argument.